### DISCUSSION TITLE
Invalid URI: The URI scheme is not valid. on submit deef Inventory

### DISCUSSION THREAD

**Discussion by TNE-germany** [abuzuhri/Amazon-SP-API-CSharp#449] [Q&A]
i try to update the quantity of a item, but always get Invalid URI.
it comes on inventory and price, with same credentials i can get the offers, so the credentials are correct.
its for the german marketplace

`var feedID = amazonConnection.Feed.SubmitFeed(xml, FeedType.POST_INVENTORY_AVAILABILITY_DATA);`
Anyone has a solution?

**Comment by RenzoF** [comment]
what are you passing in xml, a path to the XML file or the string contents?
either way, can you try the other way as well to see if it works?
